src/content/docs/apm/agents/go-agent/configuration/go-agent-configuration.mdx

```diff
@@ -1199,6 +1199,226 @@ Not all possible configuration options may be set via environment variables. The
 
+## AI monitoring [#ai-monitoring]
+
+This section includes Go agent configurations for setting up AI monitoring. 
+
+<Callout variant="important">
+Certain configurations may affect your AI monitoring data: 
+* You need to set [distributed_tracing](/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#dt-main) to `true` to capture trace and feedback data. It is turned on by default in Go agent versions 2.1.0 and higher. 
+* If `HighSecurity` is set to `true`, AI monitoring is dsiabled automatically. 
+</Callout>
+
+<CollapserGroup>
+    <Collapser
+        id="ai-monitoring-enabled"
+        title="AIMonitoring.Enabled"
+    >
+    <table>
+      <tbody>
+        <tr>
+          <th>
+            Type
+          </th>
+
+          <td>
+            Boolean
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            Default
+          </th>
+
+          <td>
+            `false`
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            [Environ variable](#environment)
+          </th>
+
+          <td>
+            `NEW_RELIC_AI_MONITORING_ENABLED`
+          </td>
+        </tr>
+
+                <tr>
+          <th>
+            Configuration function
+          </th>
+
+          <td>
+            `newrelic.ConfigAIMonitoringEnabled`
+          </td>
+        </tr>
+      </tbody>
+    </table>    
+
+    When set to `true`, enables AI monitoring.
+    </Collapser>
+    <Collapser
+        id="ai-monitoring-streaming"
+        title="AIMonitoring.Streaming.Enabled "
+    >
+    <table>
+      <tbody>
+        <tr>
+          <th>
+            Type
+          </th>
+
+          <td>
+            Boolean
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            Default
+          </th>
+
+          <td>
+            `true`
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            [Environ variable](#environment)
+          </th>
+
+          <td>
+            `NEW_RELIC_AI_MONITORING_STREAMING_ENABLED`
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            Configuration function
+          </th>
+
+          <td>
+            `newrelic.ConfigAIMonitoringStreamingEnabled`
+          </td>
+        </tr>
+      </tbody>
+    </table>    
+
+When set to `true`, enables the agent to capture streamed responses. If set to `false`, agent won't capture event data about streamed responses, but the agent can still capture metrics and spans. The span duration will end when the LLM function call exits. When set to `true`, the span duration ends when the final result is read from the stream.
+
```

Comment:
Please remove this section (documenting the `AIMonitoring.IncludeOnly` configuration option. Since the other agents don't have this configuration option, the Go agent should match the others in this case and not add the extra configuration option.